test/parallel/test-fs-promises-file-handle-write.js

```diff
@@ -26,6 +26,18 @@ async function validateWrite() {
   await fileHandle.close();
 }
 
+async function validateArrayBufferWrite() {
+  const filePathForHandle = path.resolve(tmpDir, 'tmp-arraybuffer-write.txt');
+  const fileHandle = await open(filePathForHandle, 'w+');
+  const buffer = Buffer.from('Hello world'.repeat(100), 'utf8');
+
+  await fileHandle.write(buffer.buffer, 0, buffer.length);
```

Comment:
This new signature requires specifying correct `offset` and `length` values here.

Related GH CI failures: [test-asan](https://github.com/nodejs/node/actions/runs/4206951367/jobs/7306958398) and [test-tarball-linux](https://github.com/nodejs/node/actions/runs/4206951372/jobs/7306981602)